src/clj/qbits/spandex.clj

```diff
@@ -474,18 +488,19 @@
                                request-map)
              input-ch (or input-ch (async/chan))
              output-ch (or output-ch (async/chan))
+             flush-ch (async/chan (async/sliding-buffer 1) (filter boolean))
```

Comment:
`(filter boolean)` : I guess you meant `(filter boolean?)`, in any case I not add an xform here.

Reply:
I would also allows the user to supply their own flush-ch, like we do for output-ch and input-ch, sometimes these things can be shared.

Reply:
> `(filter boolean)` : I guess you meant `(filter boolean?)`

I meant what I wrote! I often prefer truthy values that have documentation value.  E.g. `(flush! true)` might need a comment but `(flush! :before-turning-off-the-computer-for-the-weekend)` might not.

> in any case I would not add an xform here

The point of the xform is to simply drop all the false values that enter the channel.  These are no-ops, and so this makes the rest of the code easier by not having to explicitly test if the value from the channel was truthy or not.  With this transform we know that *every* value we get from the channel means that we should flush.  This makes it play really nicely with the `async/alt! call :) 

Why even support `false` values?  Well, sometimes these values are generated programmatically and if we don't have the ability to ignore them as no-ops then we force everyone else to deal with a test at the call-site.

> I would also allows the user to supply their own flush-ch, like we do for output-ch and input-ch, sometimes these things can be shared.

Done.

The commit below also removes the opinionated xform.  I can get that behavior back by sending in my own channel :) 

Thanks for the review! 👍